src/droplet.F

```diff
@@ -2083,48 +2146,51 @@ subroutine droplet_diag(rtime,nrec,zh,zf,pdata,pdata_locind,dpten,uten,vten,wten
          rad10_tmp = log10(pdata(np,prrp))
         
          !Figure out which bin it belongs to
-         if (rad10_tmp .gt. rmax10 + 0.5*dr) then
+         if (rad10_tmp .gt. rmax10 + 0.5*dhrad) then
```

Comment:
Shall we remove line 2128 to 2139 here? I saw similar calculations in the `initialize_droplet_histograms` subroutine and they did not seem to vary with different time steps.

Reply:
This is strange... Yes, you are absolutely right. But somehow these lines aren't in my local version that I had pushed. I must have messed up something when I was merging? I'd change this myself, but I don't see how. Could have this come back in when you merged gpu-opt back into this branch?

Reply:
Thanks @drichte2. I did not know why this happened, either. I just removed these lines and pushed the changes to your branch. Could you pull the latest version of your branch to your local machine and check if everything looks good to you now?

Reply:
Just pulled, looks good to me!